block.go

```diff
@@ -212,7 +213,7 @@ func writeMetaFile(dir string, meta *BlockMeta) error {
 
 	// Make any changes to the file appear atomic.
 	path := filepath.Join(dir, metaFilename)
-	tmp := path + ".tmp"
+	tmp := path + fileutil.TmpExt
```

Comment:
Instead of manually appending `fileutil.TmpExt` everywhere, would it be better to have a function call like `fileutil.TmpDir` which would take the `dir` and append `fileutil.TmpExt` to it? In case we want to change the logic of naming temp dir, this would come handy (example `ioutil.TempDir`).
```go
package fileutil

func TmpDir(dir string) string {
    return dir + TmpExt
}
```
```go
tmp := fileutil.TmpDir(path)
```

`fileutil.TmpExt` would still be present if it is needed somewhere.

Reply:
yeah I like the idea, but I prefer to be more explicit and `tmp := fileutil.TmpDir(path)` might be a bit strange. What does it do?  create a tmp dir? copies to a tmp dir ? 
refactoring is easy enough so lets keep as is and will change if needed.